addon/controllers/fd-listform-constructor.js

```diff
@@ -497,7 +518,53 @@ export default Ember.Controller.extend(FdWorkPanelToggler, {
       let treeObject = this.get('treeObjectNotUsedAttributesTree');
       treeObject.on('open_node.jstree', this._openNodeTree.bind(this));
       treeObject.on('after_close.jstree', afterCloseNodeTree.bind(this));
-    },
+    }
+  },
+
+  /**
+    This method run non ember data saved when model is loaded
+
+    @method saveOriginalData
+  */
+  originalDataInit: function () {
+    Ember.run.next(this, () => {
+      this.saveDataToOriginal();
+    });
+  },
+
+  /**
+    Save fields before changes
+
+    @method saveOriginalData
+  */
+  saveDataToOriginal: function () {
+    let originalDataString = this._getStringifyModel();
+    this.set('_originalData', originalDataString);
+  },
+
+  /**
+    Cancel form data changes
+
+    @method clearDirtyAttributes
+  */
+  clearDirtyAttributes: function () {
```

Comment:
Is there a mistake here? The name says about cleaning and in the body is made saving.